notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    request_spectra_string = f'spectra/{planet_name_formatted}/filelist/'
</pre><p>So you already knew this planet's "canonical name" and this works perfectly. But I experimented with using a non-canonical name and it did not work. Even getting rid of the "-" before the number 12 will mess this up, and least that is my finding from a sample size of a few tries ;-) This would be good to point out; even if you can search for all the different names, it seems like you MUST use the canonical name for the list of files.</p><p> </p><p>In retrospect, this would be important motivation to put at the top of this section. Otherwise as the user I don't quite see the point in creating the name_matches dictionary, because it doesn't get used again. But that's because you already knew the canonical name and defined it correctly in the first code block :-) So the first part (subsection?) is "Searching the Identifiers table of exo.MAST to find the canonical name" and then this subsection is "Searching exo.MAST for exoplanet data by name" or something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>